test-examples/jdi-bdd-no-po-tests/src/test/java/cucmberTests/test/data/TestData.java

```diff
@@ -0,0 +1,5 @@
+package cucmberTests.test.data;
+
+public class TestData {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestData' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5511453)